Sdtrig.tex

```diff
@@ -319,6 +319,37 @@ \subsection{Cache Operations}
     \end{steps}
 \end{commentary}
 
+\subsection{Invalid Virtual Addresses}
+
+For address matches without a mask, \RcsrTdataTwo must be able to hold all valid
+addresses in all supported translation modes. That means that after writing any
+of these valid addresses, the exact same value is read back, including any high
+bits.
+
+\begin{commentary}
+    If physical addresses are less than XLEN bits wide, they are zero-extended.
+    If virtual addresses are less than XLEN bits wide, they are sign-extended.
+    Be sure to have enough flops in \RcsrTdataTwo to hold all XLEN bits of the
+    full range of both physical and virtual addresses.
```

Comment:
I don't think that you actually need XLEN flops.  In the example from the email thread you need 49.  And you have to sign or zero extend based on some information in those flops.  So something more like this:
```suggestion
    Be sure to have enough flops in \RcsrTdataTwo to hold the full range of
    both physical and virtual addresses with appropriate zero or sign extension
    when read by hardware or software.
```

Reply:
Sounds good. I'm changing that.